pkg/supportbundle/load.go

```diff
@@ -65,6 +64,7 @@ func ParseSupportBundle(doc []byte, followURI bool) (*troubleshootv1beta2.Suppor
 			ObjectMeta: collector.ObjectMeta,
 			Spec: troubleshootv1beta2.SupportBundleSpec{
 				Collectors:      collector.Spec.Collectors,
+				HostCollectors:  collector.Spec.HostCollectors,
 				Analyzers:       []*troubleshootv1beta2.Analyze{},
```

Comment:
should we add `[]*troubleshootv1beta2.HostAnalyze{}` for completeness as well?

Reply:
Good catch. I have updated the PR